src/main/flight/servos.h

```diff
@@ -121,6 +121,7 @@ typedef struct servoConfig_s {
     uint16_t flaperon_throw_offset;
     uint8_t __reserved;
     uint8_t tri_unarmed_servo;              // send tail servo correction pulses even when unarmed
+    uint8_t init_servo_after_gyro_calib;    // start send PWM after gyro calibration is complete
```

Comment:
If adding this field doesn't increase `servoConfig_t`'s size (I think it will, but test it please), you have to increase the version number in `PG_REGISTER_WITH_RESET_TEMPLATE(servoConfig_t, servoConfig, PG_SERVO_CONFIG, 0);`. If you increase the version number, drop the `__reserved` reserved field.

Also, rename it to `initAfterGyro`.